kong/plugins/oidc/handler.lua

```diff
@@ -93,10 +98,16 @@ function make_oidc(oidcConfig, oidcSessionConfig)
     end
   end
 
+  -- unauth_action = pass (default)
+  -- unauth_action = nil for /api/auth/bearer/login
+  local unauth_action = "pass"
   -- grab X-Requested-With Header to see if request was from browser/ajax
-  local unauth_action = nil
   local ngx_headers = ngx.req.get_headers()
-  if ngx_headers then
+  -- @TODO: move the hard coded path to config file
+  if kong.request.get_path() == oidcConfig.idp_authentication_path then
```

Comment:
@iankoplowitzdev nice use of `kong.request.get_path()` but I think we should use `ngx.var.request_uri` for now to be consistent with the code.

Currently no kong methods are called interesting enough.

Until we learn more about lua and path of plugin would rather stay consistent with current code then add this.  This will also remove the need for the other mock.